_pages/en_US/homebrew-launcher-(browserhax-2020).txt

```diff
@@ -0,0 +1,73 @@
+---
+title: "Homebrew Launcher (BrowserHax 2020)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This method of accessing the Homebrew Launcher takes advantage of some browser oversights in the native 3DS Internet Browser.
+
+If your device already has access to the Homebrew Launcher (whether through BrowserHax, Pichaxx or another exploit), you can skip to [USM](installing-boot9strap-(hbl-usm)).
```

Comment:
It'd probably be better if there was some shoulder button warning here, even if it's *technically* not necessary for HBL access alone

Reply:
can be done